.gitignore

```diff
@@ -122,7 +122,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+aireview/src/aireview/.env
 .venv
```

Comment:
**style**: Specifying a full path to .env file is not recommended. Keep the generic .env pattern to ensure all environment files are ignored regardless of location. Add specific exceptions if needed using !pattern syntax.